questions/dl.yml

```
@@ -20,4 +20,27 @@ questions:
     - nlp
     - rnn
   diff: easy
-  ref: NULL
\ No newline at end of file
+  ref: NULL
+
+  title: Discussing dropout 
+  type: mcq
+  text: Which of the following are true regarding dropout?
+  opt:
+    - Dropout is a method of bagging training.
```

Can we modify this?:
```suggestion
    - Dropout is similar to bagging training.
```
This is more clear I guess.